testdata/go/v3/memcached-operator/README.md

```diff
@@ -48,7 +48,7 @@ make undeploy
 This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
 
 It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) 
-which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
+which provides a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster 
```

Comment:
@derekbassett thank a lot for the contribution.

But this file is a sample auto generate.
we need to change the code source and the run `make generate`

In this way, could you please
a) change it here: https://github.com/kubernetes-sigs/kubebuilder/blob/d7b802e7d7196818cc867377c17efac8f30901ad/pkg/plugins/golang/v3/scaffolds/internal/templates/readme.go#L101

b) Run `make generate`

c) Add the emoji to the title so we can properly generate the release notes. It should be : `:bug:`

d) Squash the commits and push the changes squashed ( only 1 commit )

More info: https://github.com/kubernetes-sigs/kubebuilder/blob/master/README.md

Thank you.

Reply:
/hold